includes/class-newspack-newsletters.php

```diff
@@ -447,6 +463,12 @@ public static function register_meta() {
 	public static function save( $post_id, $post, $update ) {
 		if ( ! $update ) {
 			update_post_meta( $post_id, 'template_id', -1 );
+
+			// Set "without provider" post meta if using manual.
+			$provider = self::service_provider();
+			if ( 'manual' === $provider ) {
+				update_post_meta( $post_id, 'without_provider', true );
```

Comment:
With this stored as post meta, it means that any newsletter drafts that are created while the provider is set to "manual" will always be manual-only. If I then change my provider to Mailchimp, those drafts will not be sendable via Mailchimp or any other ESP. Is this intended?

Reply:
It was intended until your comment made me think about drafts. First I thought only about published newsletters, once they are _marked as sent_ there wouldn't be a reason to handle this post with another provider.

Now I realize that a user could easily start drafting newsletters in "manual mode" before finishing setting up their provider. Because of that, I've removed the `without_provider` post meta and we now rely solely on the current provider data to display the "manual provider" features.

We are still not completely able to smoothly take over a draft with a new provider. Mailchimp, for example, always attempts to fetch the `mc_campaign_id`, which is not available for drafts that were initially made in manual provider:

![image](https://user-images.githubusercontent.com/820752/125497379-eb691ca6-1a34-4d37-8ab3-d5d68bea10e0.png)

Once you save the draft again, the sync method fixes the error message by creating the campaign and storing its ID. I'll open an issue for that.